### ISSUE TITLE
[document conversion] convert api call does not take in custom config json

### ISSUE THREAD

**Issue by feeell** [watson-developer-cloud/node-sdk#375]
According to the document conversion API docs, convert api call allows me to take in a custom config json. ([http://www.ibm.com/watson/developercloud/document-conversion/api/v1/?node#convert-document](http://www.ibm.com/watson/developercloud/document-conversion/api/v1/?node#convert-document)). 

However, according to the below code and config json I used...
```json
{
        "conversion_target": "answer_units",
        "normalized_html": {
            "exclude_tags_completely": ["script", "sup", "link"],
            "exclude_tags_keep_content": ["font", "em", "span", "strong", "code"],
            "keep_content": {
                "xpaths": [
                    "//div[@class='offer fl']"
                ]
            },
            "exclude_tag_attributes": ["EVENT_ACTIONS", "class"]
        }
    }
```

```javascript
document_conversion.convert({
        file: filestream,
        conversion_target: 'ANSWER_UNITS',
        // Use a custom configuration.
        config: config,
        content_type: file.mimetype
    }, function (err, response) {
        if (err) {
            console.error(err);
            callback(err)
        } else {
            console.log(JSON.stringify(response, null, 2));
            callback(response)
        }
    });
```

It does not take in a custom config json. 

I took a look at the source code of document conversion convert api, it parses the config like the following.
```javascript
// send the parameters in the body or as formData depending on the request
  if (params.file) {
    fixupContentType(params);
    parameters.options.formData = {
      file: params.file,
      config: {
        value: JSON.stringify(omit(params,['file', 'content_type'])),
        options: {
          contentType: 'application/json; charset=utf-8'
        }
      }
    };
  } else {
    parameters.options.body = params;
  }
```

The config.value does not seem to parse my config json. After I changed the line to value: JSON.stringify(params.config), it gets my custom json correctly. Please let me know if I am using the API wrong or my config json is wrong or it is a issue.

**Comment by nfriedly** [comment]
Hi @feeell,

This looks like a mismatch between the docs and the SDK - the SDK is expecting the config to be top-level params, where the docs (and other SDK methods from the look of it) use a standalone config param.

I'll update the SDK to make it work either way.

**Comment by nfriedly** [comment]
Should be fixed in v2.14.7 which will be automatically published to npm once the CI loop finishes
